lib/mayaUsd/fileio/translators/translatorXformable.cpp

```diff
@@ -602,6 +602,7 @@ void UsdMayaTranslatorXformable::Read(
                 "Unable to successfully decompose matrix at USD prim <%s>",
                 xformSchema.GetPath().GetText());
         }
+        resetsXformStack = true;
```

Comment:
Could you put a comment on why we need this? It will help understand what is going on in the future.